### DISCUSSION TITLE
Export flow into Python code

### DISCUSSION THREAD

**Discussion by yaruchyo** [FlowiseAI/Flowise#2166] [Ideas]
I have a question regarding the export functionality. Specifically, I'm curious to know if there's a feature that allows us to export a flow with agents into a Python file or maybe first to Json and then to python?

I've been exploring the platform and its capabilities, but I couldn't find a clear answer to this question. 
If anyone has experience or insights regarding this, I would greatly appreciate your input.

**Comment by hakan-77** [comment]
The ability to export flows and run elsewhere is pretty much the only reason why we chose Langflow over Flowise. We will reconsider our decision the day Flowise supports such functionality.

**Comment by elliotsegler** [comment]
To save others searching who land here: #275

**Comment by iaminawe** [comment]
I built a tool to do this - its not heavily tested yet and would appreciate any feedback - feel free to create issues on the repo if you encounter issues https://github.com/iaminawe/flowise-to-langchain - can generate ts or python code from your flow json.

**Reply by MohammadS3dd** [reply]
this doesnt build!

**Reply by iaminawe** [reply]
@MohammadS3dd it does now - please give it another try - thanks.
